src/server/identity/server/api_server.go

```diff
@@ -134,7 +134,7 @@ func (a *apiServer) setIdentityServerConfig(ctx context.Context, req *identity.S
 
 func (a *apiServer) GetIdentityServerConfig(ctx context.Context, req *identity.GetIdentityServerConfigRequest) (resp *identity.GetIdentityServerConfigResponse, retErr error) {
 	var config []*identity.IdentityServerConfig
-	err := a.env.DB.SelectContext(ctx, &config, "SELECT issuer, id_token_expiry, rotation_token_expiry FROM identity.config WHERE id=$1;", configKey)
+	err := a.env.DB.SelectContext(ctx, &config, "SELECT issuer, id_token_expiry AS idtokenexpiry, rotation_token_expiry AS rotationtokenexpiry FROM identity.config WHERE id=$1;", configKey)
```

Comment:
What's the deal with the lower case here?   The other one matched the struct field with the upperCamelCase.

Reply:
I'll look into this in more detail.  I think I did this one first, and my reading of the code was that it defaults to strings.ToLower to map the struct member names to db fields.  Then I had to put this aside for a week or so, forgot that, and did the other ones... which also appear to work.  I'll make it consistent though.